src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -110,28 +109,41 @@
     </table>
 
     <div class="row mb-3">
-      <div class="col-md-4 text-left d-print-none">
-        {% set numPages = (orderForViewing.products.products|length / paginationNum)|round(0, 'ceil') %}
-        <nav aria-label="Products Navigation"{% if orderForViewing.products.products|length <= paginationNum %} class="d-none"{% endif %} id="orderProductsNavPagination">
-          <ul class="pagination" id="orderProductsTablePagination" data-num-per-page="{{ paginationNum }}" data-num-pages="{{ numPages }}">
-            <li class="page-item disabled" id="orderProductsTablePaginationPrev">
-              <a class="page-link" href="javascript:void(0);" aria-label="Previous">
-                <span aria-hidden="true">&laquo;</span>
-                <span class="sr-only">Previous</span>
-              </a>
-            </li>
-            {% for numPage in 1..numPages %}
-              <li class="page-item{% if numPage==1 %} active{% endif %}"><span class="page-link" data-order-id="{{ orderForViewing.id }}" data-page="{{ numPage }}">{{ numPage }}</span></li>
-            {% endfor %}
-            <li class="page-item d-none"><span class="page-link" data-order-id="{{ orderForViewing.id }}" data-page=""></span></li>
-            <li class="page-item" id="orderProductsTablePaginationNext">
-              <a class="page-link" href="javascript:void(0);" aria-label="Next">
-                <span aria-hidden="true">&raquo;</span>
-                <span class="sr-only">Next</span>
-              </a>
-            </li>
-          </ul>
-        </nav>
+      <div class="col-md-6 text-left d-print-none">
+        <div class="order-product-pagination">
+          <div class="form-group row">
+            <label for="paginator_select_page_limit" class="col-form-label ml-3">{{ "Items per page:"|trans({}, 'Admin.Catalog.Feature') }}</label>
+            <div class="col">
+              <select id="orderProductsTablePaginationNumberSelector" class="pagination-link custom-select">
+                {% for numPageOption in paginationNumOptions %}
+                  <option value="{{ numPageOption }}"{% if numPageOption == paginationNum %} selected{% endif %}>{{ numPageOption }}</option>
+                {% endfor %}
+              </select>
+            </div>
+          </div>
+          {% set numPages = (orderForViewing.products.products|length / paginationNum)|round(0, 'ceil') %}
+          <nav aria-label="Products Navigation"{% if orderForViewing.products.products|length <= paginationNum %} class="d-none"{% endif %} id="orderProductsNavPagination">
+            <ul class="pagination" id="orderProductsTablePagination" data-num-per-page="{{ paginationNum }}" data-num-pages="{{ numPages }}">
+              <li class="page-item disabled" id="orderProductsTablePaginationPrev">
+                <a class="page-link" href="javascript:void(0);" aria-label="Previous">
```

Comment:
I've never seen this, `javascript:void(0);`, is it really useful here? 🤔 

(But you're not the one who wrote it so...)

Reply:
This is to avoid scroll moove or refresh by clicking on the link, btw you could also just kick the href off and add a `cursor: pointer;` or maybe using a button !

Reply:
It's probably ideal, this is the existing pagination I prefer not to modify it since it works for now 😅